conf/app.routes

```diff
@@ -81,6 +81,8 @@ GET        /config/search/results                       uk.gov.hmrc.cataloguefro
 GET        /config/warnings/search                      uk.gov.hmrc.cataloguefrontend.serviceconfigs.ServiceConfigsController.configWarningLanding()
 GET        /config/warnings/search/results              uk.gov.hmrc.cataloguefrontend.serviceconfigs.ServiceConfigsController.configWarningResults()
 
+GET        /estimated-cost-explorer                     uk.gov.hmrc.cataloguefrontend.costs.CostsSummaryController.onPageLoad(team: Option[String] ?= None)
```

Comment:
Do we need `estimated` in the URL?  It doesn't look like we're disambiguating with another cost-explorer - so I'd be tempted to leave this to an explanation on the page itself.

Reply:
Should the controller be called `CostExplorerController` - if that's how we're thinking of it it would make finding it easier